src/actions/websocket.js

```diff
@@ -560,7 +560,8 @@ function handleChannelDeletedEvent(msg, dispatch, getState) {
             dispatch({type: ChannelTypes.SELECT_CHANNEL, data: channelId}, getState);
             EventEmitter.emit(General.DEFAULT_CHANNEL, '');
         }
-        dispatch({type: ChannelTypes.RECEIVED_CHANNEL_DELETED, data: msg.data.channel_id}, getState);
+
+        dispatch({type: ChannelTypes.RECEIVED_CHANNEL_DELETED, data: {id: msg.data.channel_id, team_id: msg.data.team_id, deleteAt: msg.data.delete_at}}, getState);
```

Comment:
@enahum Add you so that you're aware that this may have cause bugs for you on mobile.

//cc @hmhealey

Reply:
What kind of bugs? Didn’t you try it? If that is the case I’m not comfortable to merge this unless it has been handled on mobile.

I do believe we are all responsible for all the clients

Reply:
@enahum I worded that poorly. It may FIX bugs that currently exist.

Reply:
Ohhhhhhh that is totally different :)